### PR TITLE
Add Russian translation file

### PR DESCRIPTION
**Pull request by bastshoes**

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @bastshoes, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant-polymer#525).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: translations/ru.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "language": {
+    "en": "Русский"
```
Comment (c727) [inline translations/ru.json]: also use "ru" here

File: translations/ru.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  },
+  "panel": {
+    "config": "Настройка",
```
Comment (andrey-git) [inline translations/ru.json]: "Настройки"?

File: translations/ru.json (hunk 1 above)
Comment (bastshoes) [reply translations/ru.json]: Fixed

File: translations/ru.json (hunk 2 above)
Comment (bastshoes) [reply translations/ru.json]: Согласен.

**Review by andrey-git — APPROVED** [review]

**Comment by moskovskiy82** [comment]
What about changing  +    "history": "История",
To "События" which better reflects what it is.

Also as a the suggestion "states": "Обзор" doesn't sound right. Although can't come up yet with a better alternative - Состояние, Статус, Обстановка all sound as weird.


 +    "log_out": "Выход"

**Comment by andrey-git** [comment]
@moskovskiy82 I think События is too similar to Журнал which we use for logbook.
*states* was changed to *overview* in English, so I think Обзор is a good match now.

**Comment by bastshoes** [comment]
History-История. It's a well established translation and it's understandable and usual for users.

**Pull request merged**